docs/content/guidelines/usage.mdx

```diff
@@ -5,74 +5,93 @@ title: Usage guidelines
 import {Caption, Do, Dont} from '@primer/gatsby-theme-doctocat'
 import {Grid, Flex, Box} from '@primer/components'
 
-# Octicons usage guidelines
+## General guidelines
 
-The following guidelines are to be used as a reference for how to use octicons.
+1. Octicons are used to _emphasize_ meaning, not as a replacement.
+2. Adding an octicon is intentional.
+3. An icon is need. Not everything needs an icon: consider a text label before adding an icon.
+4. Octicons are kept at their 1:1 aspect ratio.
 
-## General guidelines:
+## Sizing
 
-These are general guidelines to follow when using octicons across various platforms:
+Octicons are available in two sizes: 16px and 24px. They should only be used at their dedicated sizes to maintain consistency in stroke width and legibility.
 
-1. Octicons should be used to _emphasize_ meaning, not replace.
-2. Adding an octicon should be intentional.
-3. Not everything needs an icon. Consider a text label before adding an icon.
-4. Octicons should be kept at their 1:1 aspect ratio
+<img width="960" alt="octicons-scaling" src="https://user-images.githubusercontent.com/10384315/103715870-87eb1d80-4f76-11eb-8b54-c25ba09ecb5f.png" />
```

Comment:
It isn't clear to me what this image is trying to communicate. If we're trying to say "avoid scaling 16px and 24px icons," then a do/don't example might be more clear.

Reply:
I wanted to communicate the change in stroke weight and legibility. I can change this do a do/don't example though.